projects/admin/src/assets/rero-ils-ui/admin/i18n/ar.json

```diff
@@ -113,6 +113,7 @@
   "Format": "Format",
   "Gender": "النوع",
   "Get": "Get",
+  "Global shortcuts": "Global shortcuts",
```

Comment:
we do not have to extract translation anymore, I think. Please check with @iGormilhit.